packages/snap/src/index.ts

```diff
@@ -143,29 +141,62 @@ async function sendTransaction(tx: CeloTransactionRequest, wallet: CeloWallet) {
   return txResponse.wait()
 }
 
-async function getBIP44Node(index: number = 0): Promise<BIP44Node> {
-  // Get the bip44 node corresponding to the path m/44'/52752'/0'/0
-  const bip44Node = await snap.request({
+async function getKeyPair(snap: SnapsGlobalObject, address?: string, addressIndex: number = 0): Promise<KeyPair> {
+  const derivationPath = "m/44'/60'/0'/0"; //Todo - read from state config
+  let extendedPrivateKey;
+  let _addressIndex = 0;
+  const MAX_SEARCH_DEPTH = 50;
```

Comment:
For now .. we stop constructing the key tree until the 50th loop as a limit.

Reply:
thag seems good to me,